client/packages/invoices/src/Returns/InboundDetailView/columns.ts

```diff
@@ -151,6 +152,26 @@ export const useInboundReturnColumns = ({
       //         ]),
       //     },
       //   ],
+      {
+        key: 'packUnit',
+        label: 'label.pack',
+        sortable: false,
+        Cell: PackVariantCell({
+          getItemId: row => {
+            if ('lines' in row) return '';
+            else return row?.item?.id;
+          },
+          getPackSizes: row => {
+            if ('lines' in row) return row.lines.map(l => l.packSize ?? 1);
+            else return [row?.packSize ?? 1];
+          },
+          getUnitName: row => {
+            if ('lines' in row) return null;
```

Comment:
```suggestion
            if ('lines' in row) return row.lines[0]?.item?.unitName ?? null;
```
Any reason this isn't the same as Outbound Return? When the IR is not grouped by item:
![Screenshot 2024-04-05 at 8 48 50 AM](https://github.com/msupply-foundation/open-msupply/assets/55115239/9185a4d5-1414-4143-a6a5-23b68c062af2)

When grouped by item:
![Screenshot 2024-04-05 at 8 48 42 AM](https://github.com/msupply-foundation/open-msupply/assets/55115239/cdac01f6-6e5c-4f22-a110-78b325fde96b)

When not showing [multiple] I think it should still show the unit?

Reply:
Yeah, except this is how it is in Inbound Shipment -- I just copied that. Should we leave it for now and change them both once we know exactly what the requirement is?

Reply:
Just checked with Andrei - says both should show the unit name 🤓

Reply:
Righto, made that change to both. Thanks a81185b38